releaseNotes/master.md

```diff
@@ -11,10 +11,13 @@ ${REL_VER_DATE}
   - **Updated AWS SDK in deploy jobs**: Deploy jobs in Shippable Assembly Lines now use a more recent version of the AWS SDK, with support for `healthCheck` in task definitions.
   - **shipctl notify support for IRC and Windows**: `shipctl notify` can now be used with IRC notification resources, and is also fully supported on Windows nodes. See [shipctl documentation](http://docs.shippable.com/platform/tutorial/workflow/using-shipctl/#notify) for more details.
   - **Enhanced console log handling**: The enhanced, non-blocking approach towards handling console logs will now be the default handler instead of being opt-in. You can expect an improvement in the stability and speed of your builds, especially if it produces a significant amount of console logs.
+  - **Enhanced Release notes**: Now the latest 10 release notes are available at https://app.shippable.com/changelog.
```

Comment:
add that this can be accessed by clicking the newspaper icon on the top right of the app